### ISSUE TITLE
[Admin - partie conférences]  erreur formulaire

### ISSUE THREAD

**Issue by vinceAmstoutz** [afup/web#1277]
En écrivant les tests fonctionnels la dernière fois je me suis aperçu que quand on veut ajouter une conférence via le BO et que l'on renseigne uniquement les champs requis on a cette erreur

![image](https://user-images.githubusercontent.com/46444652/224552229-d9acaf1d-a86e-4cc0-8e91-cd71fa4bbc3b.png)

**Cependant, quand on remplis ces 3 éléments comme s'ils étaient obligatoire (alors que ce n'est pas spécifié à l'affichage) ca marche !**
![image](https://user-images.githubusercontent.com/46444652/224552296-0dbe0722-7af4-49d3-92ca-d9d54af5377b.png)

@agallou je peux passer ces champs en obligatoire ?

**Comment by agallou** [comment]
oui, ça serait mieux si ces champs seraient obligatoires. Bien vu, et merci à toi.

**Comment by vinceAmstoutz** [comment]
Ok je ferai ca alors :)
